### ISSUE TITLE
Facade name

### ISSUE THREAD

**Issue by IlCallo** [hisorange/browser-detect#80]
Currently the facade is called `Facade`, but in Laravel core and all other packages Facades are always defined into a folder `Facades` taking the name of the alias.

Also, by convention the namespace is always with capital letter, so `Hisorange` instead of `hisorange`.

Is it possible to uniform this to all other Laravel packages? I can make a PR if you wish.

**Comment by hisorange** [comment]
Thanks for the heads up, but I chose the _hisorange_ namespace because I use this name in lowercase everywhere, also the facade got it's name because this is what it serves, I know about those conventions, but I find it weird to move something under a namespace like hisorange\BrowserDetect\Facades\Browser when there is no other facade and since it's a facade you can name it anyway. 

TLDR; I am kinda crazy with naming and spent a lot of time deciding the current ones :D

Thanks!
